tests/test_models.py

```diff
@@ -3904,6 +3897,91 @@ def test_node_scale_with_deleted_parent(self):
         link.save()
         assert_equal(link.node_scale(node), -40)
 
+class TestDraftRegistration(OsfTestCase):
+
+    def setUp(self, *args, **kwargs):
+        super(TestDraftRegistration, self).setUp(*args, **kwargs)
+
+        self.user = AuthUserFactory()
+        self.auth = self.user.auth
+        self.node = ProjectFactory(creator=self.user)
+
+        MetaSchema.remove()
+        ensure_schemas()
+        self.meta_schema = MetaSchema.find_one(
+            Q('name', 'eq', 'Open-Ended Registration') &
+            Q('schema_version', 'eq', 1)
+        )
+        self.draft = DraftRegistration(
+            initiator=self.user,
+            branched_from=self.node,
+            registration_schema=self.meta_schema,
+            registration_metadata={
+                'summary': {'value': 'Some airy'}
+            }
+        )
+        self.draft.save()
+
+    def test_factory(self):
```

Comment:
This test seems to cover several different scenarios. It may be worth breaking them up into separate tests with more descriptive names. Additionally, can some of this logic be used with the setup method to reduce redundancy?